tsconfig.base.json

```diff
@@ -9,7 +9,7 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "target": "es2015",
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
going back to the old days?

Reply:
Using the CommonJS module system avoids the building issue Ragith and I were having with TypeORM.

Reply:
I see, alright let's make this problem of the future, and implement some features. this is no biggie.

Reply:
I will be moving this decleration of module system into the mull-api's tsconfig, to avoid affecting other apps.